apps/judicial-system/backend/src/app/modules/event-log/models/eventLog.model.ts

```diff
@@ -18,12 +18,39 @@ import { Case } from '../../case/models/case.model'
   timestamps: false,
 })
 export class EventLog extends Model {
-  static caseSentToCourtEvent(eventLogs?: EventLog[]) {
-    return eventLogs?.find(
-      (eventLog) =>
-        eventLog.eventType === EventType.CASE_SENT_TO_COURT ||
-        eventLog.eventType === EventType.INDICTMENT_CONFIRMED,
-    )
+  static getEventLogByEventType(
```

Comment:
NIT: Apart from the user role, this helper identical to the helper in `DefendantEventLog`. Is it possible to extract shared logic to a suitable place in the code? where we can also add tests?

Reply:
Actually, the two methods handle different enum types, so I don't think it would be a good idea to mix the two.